test/services/setup_creator.js

```diff
@@ -134,27 +135,33 @@ describe('Setup Creator', () => {
     });
   });
 
-  describe('copyChainJson', () => {
-    const networkName = 'dev';
-    const chainJsonContents = `{"name": "${networkName}"}`;
-    const srcChainJsonPath = `${testInputDir}chain_files/${networkName}.json`;
+  describe('fetchChainJson', () => {
+    const chainSpecUrl = 'https://chainspec.ambrosus-dev.com/';
+    const chainJsonContent = '{"name": "dev"}';
     const destChainJsonPath = `${testOutputDir}chain.json`;
 
-    beforeEach(async () => {
-      await makeDirectory(`${testInputDir}chain_files`);
-      await writeFile(srcChainJsonPath, chainJsonContents);
-    });
-
     afterEach(async () => {
+      nock.cleanAll();
+      nock.enableNetConnect();
```

Comment:
You can safely remove this line. You're not changing the default behavior of nock, so no need to restore it.